docs/user_guide/provisioning_tool.rst

```diff
@@ -4,7 +4,13 @@
 Provisioning in NVIDIA FLARE
 ############################
 
-The purpose of provisioning in NVIDIA FLARE is to generate mutual-trusted system-wide configurations for all participants
+A necessary first step in establishing a federation is provisioning, in which the Open Provision API and its builder
```

Comment:
When reading this line, I will try to search what is "Open Provision API".

Maybe we should say for provisioning we have a provision API provided in NVFlare, and we call it Open Provision API?